zerver/lib/upload.py

```diff
@@ -193,12 +192,20 @@ def get_total_uploads_size_for_user(user: UserProfile) -> int:
         total_quota = 0
     return total_quota
 
-def within_upload_quota(user: UserProfile, uploaded_file_size: int) -> bool:
-    total_quota = get_total_uploads_size_for_user(user)
-    if (total_quota + uploaded_file_size > user.quota):
-        return False
-    else:
-        return True
+def currently_used_upload_space(realm: Realm) -> int:
+    total_quota = Attachment.objects.filter(realm=realm).aggregate(Sum('size'))['size__sum']
```

Comment:
I would s/total_quota/used_space
quota refers to the amount you are allowed to use, rather than the amount used.

Reply:
bump on this